client/reader/search-stream/blank.jsx

```diff
@@ -1,5 +1,27 @@
-import React from 'react';
+const React = require( 'react' );
 
-export default function BlankContent() {
-	return <p />;
-}
+const BlankContent = React.createClass( {
+
+	propTypes: {
+		suggestions: React.PropTypes.array
+	},
+
+	shouldComponentUpdate: function() {
+		return false;
```

Comment:
this isn't right, as you'd want this component to re-render when the set of suggestions passed in changes.